books/projects/aleobft/dynamic/definition.lisp

```diff
@@ -32,9 +32,12 @@
      and the possible transitions by which
      an event takes a state to a new state
      (the events `label' the transitions between states).")
-   (xdoc::p
-    "The definition of our labeled state transition is executable,
-     in the sense explained in @(see transitions), useful as a simulator."))
+; re-add this once we have attachments for the constrained functions.
+; See 'transitions' for more detail.
+;   (xdoc::p
+;    "The definition of our labeled state transition is executable,
+;     in the sense explained in @(see transitions), useful as a simulator.")
+   )
```

Comment:
I would remove these commented lines because we may not actually add these attachments.